doc/source/user_guide/usability.rst

```diff
@@ -128,9 +134,11 @@ Examples
    ...
    >>>
    >> # Wildcard pattern search within a specific API object
-   >>> pyfluent.search("local*", api_path="mesh_interfaces")
-   <solver_session>.setup.mesh_interfaces.interface["<name>"].local_absolute_mapped_tolerance (Parameter)
-   <solver_session>.setup.mesh_interfaces.interface["<name>"].local_relative_mapped_tolerance (Parameter)
+   >>> pyfluent.search("iter*", api_path="<solver_session>.parallel.multidomain") 
+   <solver_session>.parallel.multidomain.conjugate_heat_transfer.set.coupling.iter_per_coupling_count (Parameter)
+   <solver_session>.parallel.multidomain.conjugate_heat_transfer.set.coupling.single_session_coupling.iteration (Parameter)
+   <solver_session>.parallel.multidomain.solve.dual_time_iterate (Command)
+   <solver_session>.parallel.multidomain.solve.iterate (Command)
    >>>
```

Comment:
@hpohekar This became a very strong feature overall.

Reply:
Yes, I agree. Thanks a lot for the suggestions and feedback given by the every team member.